src/xcode/ENA/ENA/Source/Scenes/Settings/SettingsCoordinator.swift

```diff
@@ -135,7 +135,6 @@ class SettingsCoordinator: ENStateHandlerUpdating {
 		let dataDonationViewController = DataDonationViewController(viewModel: viewModel)
 		dataDonationViewController.navigationItem.title = AppStrings.DataDonation.Info.title
 		dataDonationViewController.navigationItem.largeTitleDisplayMode = .always
-		dataDonationViewController.navigationController?.navigationBar.prefersLargeTitles = true
```

Comment:
What does this deletion change?

Reply:
Actually, this line had no effect, because the dataDonationViewController has no navigation controller before it is pushed on the stack. I removed the line mainly because it is / was confusing and misleading.

By the way, the other two lines above are duplicated in DataDonationViewController.viewDidLoad(). Currently this duplication is aligned, but could easily run out of synch in future. For sake of maintainability, I suggest to remove those lines from one or the other location (or even move them into DataDonationViewController.viewWillAppear if you prefer that).

Reply:
The duplication was also something I stumbled over. I would also prefer to remove it at one place.

Reply:
> The duplication was also something I stumbled over. I would also prefer to remove it at one place.

Done.